values.yaml

```diff
@@ -777,6 +777,12 @@ database:
     #  requests:
     #    memory: 256Mi
     #    cpu: 100m
+    # The timeout used in livenessProbe; 1 to 5 seconds
+    livenessProbe:
```

Comment:
Could you add the same configuration for all other components so that they are consistent?

Reply:
What other components? I want to touch the internal database only.

Reply:
@ywk253100 : what other components to you have in mind? Yes, the probes have more parameters and as a regular harbor user, I also like to have things be consistent.

Reply:
@ywk253100 : Can you please comment what you need for your approval? I'm a bit lost. I can add more configurable parameters, but which ones? My PR is for liveness/readinessProbes for the build-in database only. I can also use other parameter names if you find them not suitable. Best, Peter